view credential page

jrsarge opened the pull request:
Created a view credential page and created functionality to auto populate the credential you wanted to issue if you navigated  to the issue page from the view credential page.

NOTES:
-Someone teach me how to take the port changes on the docker file out of my pull request
-Kinda hacked it to get the credential auto populated on the issue page bc I couldn't figure out how it worked. If someone could explain that to me I would appreciate it
-When making the table I realized it might not even be possible to edit a saved credential bc it's on the chain. We can talk about it more at hackathon

jrsarge committed:
view credential page + auto-populate issue

kahuku commented on client/src/pages/IssueCredential.tsx:
```diff
@@ -16,26 +16,32 @@ import {
   useEffect(() => {
+    console.log("credentialType:", credentialType); // Add logging
+    if (!credentialType) {
+      setError("Credential type is missing from the URL");
```
I don't think we want to have an error if there isn't a credential type in the URL (like when someone would come from the dashboard page rather than the create credential page)

jrsarge replied:
True, true. I also changed the param to optional. Thank you for the catch drew!

kahuku commented:
I can tell you how to fix the docker thing via the command line but I don't know github desktop haha. Now that you've committed and pushed though, you could also change it back and push it, then change it back to what you need it to and not push it. Basically you just need to not "stage" that file's changes. Via CLI you'd go git add [files], git commit, git push, and you'd just want to not include that file in the git add command.

Good plan with the hackathon. I have no idea what we want to try for there. 

Autofilling is a nice touch, way to think of that. Looks like you just included it as a URl parameter (see my comment above, we might want to make that an optional query param instead)

tannergwood commented:
Yea the autofill is sick! Maybe the edit can be used to edit the credential details that are stored with pinata, idk if we can actually edit that or not.

jrsarge committed:
Fixes

Changed port back to 5433 in docker file, removed param error message, changed query param to optional
